src/DependencyInjection/SymfonycastsSassExtension.php

```diff
@@ -64,6 +69,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->info('The Sass binary to use')
                     ->defaultNull()
                     ->end()
+                ->scalarNode('embed_sourcemap')
+                    ->info('Whether to embed the sourcemap in the compiled CSS')
```

Comment:
What do you think about mentioning this in the config?
```suggestion
                    ->info('Whether to embed the sourcemap in the compiled CSS. By default, enabled only when the debug mode is on.')
```